src/Downloader.ts

```diff
@@ -17,7 +17,6 @@ import https from 'https'
 
```

Comment:
I keep advising this, for nobody the separation of duties between the multiple module is clear.

Reply:
I don't have anything to add about your description of Downloader class, so I included it at the top of the class definition instead of the method description.